src/main/java/text/Print.java

```diff
@@ -0,0 +1,42 @@
+package text;
+
+import task.TaskManager;
+
+public class Print {
```

Comment:
![image](https://user-images.githubusercontent.com/35750468/93706715-7adb2780-fb5b-11ea-9b1e-829a8c6c6fed.png)

take note of your naming!